src/Commands/InstallCommand.php

```diff
@@ -69,7 +70,19 @@ public function handle(Filesystem $filesystem)
         $this->call('hook:setup');
 
         $this->info('Publishing the Voyager assets, database, language, and config files');
-        $this->call('vendor:publish', ['--provider' => VoyagerServiceProvider::class]);
+
+        //Publish only relevant resources on install
+        $tags = ['voyager_assets', 'seeds', 'demo_content', 'config', 'migrations', 'lang'];
+
+        if (app()->version() >= 5.4) {
+            $tags = array_diff($tags, ['migrations']);
```

Comment:
This is some REALLY weird logic... you define the full array, then remove an entry... then add it back.  This needs to be changed